test_scripts/Defects/4_6/1398_RegisterAppInterface_SDL_returns_DISALLOWED_instead_of_DUPLICATE_NAME.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1398
+--
+-- Precondition:
+-- 1) SDL and HMI are running.
+-- 2) Mobile App is registered to SDL with appName=1, appID=1
+
+-- Description:
+-- Steps to reproduce:
+-- 1) Try to register new app with the same: appName=1 and appID=1
+-- 2) Try to register new app with the same: appName=2 and appID=1
```

Comment:
This step is absent in script and i think that it is redundant in description. Defect description is related to case1.